pkgs/development/interpreters/perl/default.nix

```diff
@@ -152,6 +152,7 @@ let
     meta = {
       homepage = https://www.perl.org/;
       description = "The standard implementation of the Perl 5 programmming language";
+      license = licenses.gpl1;
```

Comment:
Per https://dev.perl.org/licenses/ you can choose gpl v1 or any later version. It is really gpl + clarifications by the author, however, so maybe consider using the Artistic License which makes those clarifications explicit.

Reply:
Should I make it just make it `[ gpl1Plus artistic1 ]`? (Perl 6 would be `artistic2`?)

Reply:
I would just use Artistic, because it expresses most clearly the intent of the author, but I'm happy to leave that to your judgment.

Reply:
To clarify, I'd be happy to leave it as is, just wanted to mention this as a potential consideration, not a blocker or anything.

Reply:
I can see your point here. We can change it to Artistic.